datafusion/optimizer/src/common_subexpr_eliminate.rs

```diff
@@ -885,15 +876,15 @@ mod test {
         )?;
 
         let expected = vec![
-            (8, "{(SUM(a + Int32(1)) - AVG(c)) * Int32(2)|{Int32(2)}|{SUM(a + Int32(1)) - AVG(c)|{AVG(c)|{c}}|{SUM(a + Int32(1))|{a + Int32(1)|{Int32(1)}|{a}}}}}"),
-            (6, "{SUM(a + Int32(1)) - AVG(c)|{AVG(c)|{c}}|{SUM(a + Int32(1))|{a + Int32(1)|{Int32(1)}|{a}}}}"),
+            (8, "#{(SUM(a + Int32(1)) - AVG(c)) * Int32(2)}"),
```

Comment:
`IdArray` shouldn't change by this PR. Actually, new fields can be added to the tuples but the identifier field should remain as it is.